conda-recipes/mxnet-feedstock/.ci_support/linux_ppc64le_python3.6.yaml

```diff
@@ -0,0 +1,16 @@
+c_compiler:
+- gcc
+c_compiler_version:
+- '7'
+channel_sources:
+- conda-forge,defaults
```

Comment:
Note this order specifies priority. Does it still work if you put `defaults` first?